dockers/docker-sonic-mgmt-framework/base_image_files/sonic-launch-shell

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+# Launch the "preferred" shell based on the groups that the user belongs to.
+# The preferred shell is either:
+#    /bin/bash:          If the user belongs to group "admin"
+#    /usr/bin/sonic-cli: Otherwise
+
+# Uncomment "set -x" for debugging
+#set -x
+
+if /usr/bin/id -Gn | tr " " "\n" | /bin/grep -qw "^admin$"; then
+    exec /bin/bash "$@"
+else
+    exec /usr/bin/sonic-cli
```

Comment:
Current sonic-cli does not provide all commands supported, like the Click commands. Hence, it is not a good idea to launch sonic-cli for remote user. Instead, bash shall be launched where the user can perform CRUD operations via Click commands.

Also, for TACACS, bash is being launched now. Hence to maintain consistency across all remote users, we shall retain TACACS shell launch behavior for RADIUS too.

https://github.com/sonic-net/sonic-buildimage/pull/13141 addressing the proper approach.